gateway/src/apicast/http_proxy.lua

```diff
@@ -118,6 +140,11 @@ local function forward_https_request(proxy_uri, uri, skip_https_connect)
         end
     end
 
+    -- The whole request is buffered in the memory so remove the Transfer-Encoding: chunked
+    if request.headers["Transfer-Encoding"] == "chunked" then
```

Comment:
We should make it very clear here that this is a short-term fix and when we finally add support for chunked requests to then remove this logic.